tests/behat/src/FixtureContext.php

```diff
@@ -508,4 +509,31 @@ public function iScrollTheEditorDetailsPanelToTheTop()
         $script = "document.querySelector('.editor__details fieldset').scrollTo(0, 0);";
         $this->getMainContext()->getSession()->executeScript($script);
     }
+
+    /**
+     * Example: Given a maximum file size is 5k
+     *
+     * @Given /^a maximum file size is "([^"]+)"$/
+     * @param string $size Max file size
+     */
+    public function stepCreateMaximumFileSizeStep($size): void
+    {
+        $uploadValidatorClass = Upload_Validator::class;
+        $config = <<<YAML
+---
+name: fileallowedsize
+---
+
+$uploadValidatorClass:
```

Comment:
```suggestion
SilverStripe\Assets\Upload_Validator:
```
This isn't dynamic - it will never change. There's no need to inject it into the string like that.
You _might_ need to escape the backslashes.